source/services/SentryMonitoringService.ts

```diff
@@ -0,0 +1,21 @@
+import * as Sentry from "@sentry/react-native";
+import env from "react-native-config";
+
+import type { MonitoringService } from "./MonitoringService";
+
+const sentryMonitoringService: MonitoringService = {
+  init: () => {
+    Sentry.init({
+      dsn: env.SENTRY_DSN,
+      // Set tracesSampleRate to 1.0 to capture 100% of transactions for performance monitoring.
```

Comment:
Comment necessary? Doesn't seem to add much value.

Reply:
Will keep it for now. Think it could be meaningful if we need to change the sampling rate.